src/mendeleyMain.vb

```diff
@@ -257,8 +257,8 @@ End Function
 Function apiUndoManualFormat(fieldCode As String) As String
     apiUndoManualFormat = mendeleyApiCall("citation_undoManualFormat", fieldCode)
 End Function
-Function apiCitationChoose(hintText As String) As String
-    apiCitationChoose = mendeleyApiCall("citation_choose_interactive", hintText)
+Function apiCitationChoose(hintText As String,CitTemp as String,cpage as Integer) As String
```

Comment:
We should be more consistent in our local variable naming.  Camel case as Adam calls it should be preferred.  (Start with lower case then capitalise the following words. e.g. anExampleOfCamelCase)